src/runtimes/python3/bootstrap

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -eu
+
+# Use the system installed python3
+PYTHONBIN="$( command -v python3 || command -v python || echo '' )"
+if [ -z "$PYTHONBIN"  ]; then
```

Comment:
We should probably add a version check, similar to the `bootstrap.ts` file's logic.

Reply:
@TooTallNate Good idea. Is there a POSIX compliant `string.startsWith()`?